### ISSUE TITLE
Increasing max channels for NodeProxies (i.e. SystemSynthDefs interface and documentation)

### ISSUE THREAD

**Issue by jamshark70** [supercollider/supercollider#5724]
## Motivation

Just seen on Facebook:

> I'm getting an error, I haven't seen yet, googling didn't give any helpful results. The error is: "Can't set more than 16 channels with current setup in SystemSynthDefs." I got it after creating an Ndef which contained an array with 28 channels.

- If the user tries to open SystemSynthDefs help, they get: "This class is missing documentation."
- If we look at the class definition and guess that increasing `numChannels` seems promising, then they will find that additional system_link synthdefs are *not* created.

The action of creating the SynthDefs is folded into the StartUp function, and is not exposed as a method. So, AFAICS, there are only three ways to increase the number of channels for SystemSynthDefs:

1. Startup file (because the startup file executes before StartUp functions).
2. Or, create a class and set numChannels in `*initClass`.
3. Or, `SystemSynthDefs.numChannels = 28; StartUp.run;` which a/ is amusing :confused: and b/ reruns *every* startup function, which is at best redundant.

This is somewhat less than user-friendly.

## Description of Proposed Feature

1. Add a class method `*addSynthDefs` to SystemSynthDefs, and move all of the SynthDef loops into it.
2. Replace the startup function with `{ this.addSynthDefs }`.
3. Add `*numChannels_ { |value| numChannels = value; this.addSynthDefs }`.
4. Documentation.

## Plan for Implementation

I'd guess 5 minutes for code changes, half an hour for the help file?

**Comment by jamshark70** [comment]
While updating the class, it would also be good in `numChannels_` to convert the user value to an integer. The same user on Facebook just broke it by putting `SystemSynthDefs.numChannels = 2**5` into startup.scd.

**Comment by telephon** [comment]
yes, at least that would be kind of polite. I think it is only a problem because `"system_link_audio_" ++ i` differs when `i` is a float. One could also do the conversion there:`(1..numChannels.asInteger)`.
